bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/discovery/HueBridgeUPNPDiscoveryParticipant.java

```diff
@@ -110,13 +110,13 @@ public Set<ThingTypeUID> getSupportedThingTypeUIDs() {
         }
         URL baseUrl = details.getBaseURL();
         String serialNumber = details.getSerialNumber();
-        String label = details.getFriendlyName();
+        String label = String.format(DISCOVERY_LABEL_PATTERN, baseUrl.getHost());
```

Comment:
Sorry, but the `baseUrl` null check is in line 119, so potential NPE here.

Reply:
Arghhhh
I am going to fix that.

Reply:
Done